air-quality-backend/tests/etl/forecast/forecast_data_test.py

```diff
@@ -70,7 +70,14 @@ def test__get_pollutant_data_for_lat_long(latitude: float, longitude: float, exp
         data_vars=dict(no2=input_data, t=t),
     )
     forecast_data = ForecastData(single_level, multi_level)
-    result = forecast_data.get_pollutant_data_for_lat_long(
-        latitude, longitude, PollutantType.NITROGEN_DIOXIDE
+    location = {
+        "name": "test",
+        "type": "test",
+        "latitude": latitude,
+        "longitude": longitude,
+    }
+    result = forecast_data.get_pollutant_data_for_locations(
+        [location],
```

Comment:
Possible to add a test with multiple locations and pollutants (or update this one)?